kubernetes/manifests/_/rbac/cluster-roles/vipyrsec.yaml

```diff
@@ -0,0 +1,40 @@
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+
+metadata:
+  name: vipyrsec
+
+rules:
+  - apiGroups:
+      - ''
+    resources:
+      - pods
+      - services
+      - configmaps
+      - secrets
```

Comment:
Regular Vipyrsec users probably shouldn't have vis of secrets. We can reserve that for core-devs and admins at bare minimum.